packages/dashboard/src/pages/Wallets/hooks/useERC20TokensDetailed.ts

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
🤔Why doesn't it move to web3-shared? And I think the existing Hook should be able to meet your needs and there should be no need to create a new hook

Reply:
I think this hook is work for dashboard UI. It wrapping the useERC20TokensDetailedFromTokenLists, an improved TokenList component code readability.